src/Microsoft.DotNet.Cli.Utils/ProjectContextExtensions.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+using System.IO;
+using Microsoft.DotNet.ProjectModel;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    internal static class ProjectContextExtensions
+    {
+        public static string ProjectName(this ProjectContext context) => context.RootProject.Identity.Name;
+
+        public static string GetOutputPath(this ProjectContext context, string configuration, string currentOutputPath)
```

Comment:
I copied this code straight from Compiler. I did not quite understand why the two methods exist, so I went with what Compile used to not risk modifying Compile's behaviour.

Can you please open a new issue to fix this duplication?